analyzers/its/expected/Roslyn.1.3.1/Roslyn.1.3.1--net452-S2259.json

```diff
@@ -0,0 +1,43 @@
+{
```

Comment:
Only this file is relevant. The other issues are irrelevant noise. I could try to address some for price of adding more noise to OldRoslyn.cs. No strong opinion on my side